includes/wp-cli.php

```diff
@@ -29,6 +29,7 @@ function prepare_environment() {
 
 	// Set DOING_CRON when appropriate
 	if ( isset( $cmd[1] ) && 'orchestrate' === $cmd[1] ) {
+		@ini_set( 'display_errors', '0' );
```

Comment:
Minor: but it would be useful to have an inline comment explaining why this is needed.

Maybe worth adding `@codingStandardsIgnore` as well?